haskell-src/Concordium/Types/Parameters.hs

```diff
@@ -544,7 +558,7 @@ $( singletons
         -- \|Consensus parameters version.
         data ConsensusParametersVersion
             = ConsensusParametersVersion0 -- \^Election difficulty
-            | ConsensusParametersVersion1 -- \^Timeout parameters, block energy limit, min block time
+            | ConsensusParametersVersion1 -- \^Timeout parameters, block energy limit, min block time, finalization committee parameters
```

Comment:
Should the finalization committee parameters be mentioned here? It seems that they are not a part of the consensus parameters.